environment/src/main/java/com/sequenceiq/environment/environment/service/freeipa/FreeIpaService.java

```diff
@@ -77,6 +77,11 @@ public DescribeFreeIpaResponse create(CreateFreeIpaRequest createFreeIpaRequest)
         try {
             return freeIpaV1Endpoint.create(createFreeIpaRequest);
         } catch (WebApplicationException e) {
+            ExceptionResponse exceptionResponse = e.getResponse().readEntity(ExceptionResponse.class);
+            if (exceptionResponse != null && exceptionResponse.getMessage() != null &&
+                    exceptionResponse.getMessage().contains("already exist")) {
+                return freeIpaV1Endpoint.describe(createFreeIpaRequest.getEnvironmentCrn());
+            }
```

Comment:
this seems very fragile for me, as if we change the error message on the FMS side this code is broken. I would call a list or describe first and if freeipa is not there then call create
this is how it's done between datalake and core service: https://github.com/hortonworks/cloudbreak/blob/d5db3c1dc6ac46c0bc180044e3867df834585122/datalake/src/main/java/com/sequenceiq/datalake/service/sdx/ProvisionerService.java#L173-L179